setup.py

```diff
@@ -38,7 +38,8 @@ def _load_description() -> str:
         "fastapi",
         "grpcio",
         "importlib-metadata;python_version<'3.8'",
-        "numpy",
+        # numba 0.55.0 requires: numpy <1.22 (for Shap / alibi)
+        "numpy==1.21.5",
```

Comment:
Just wondering, would it work if we pinned the version of `numpy` in the dependencies of the Alibi runtime package?

Reply:
I didnt work for some reason but let me double check.